main.go

```diff
@@ -35,6 +35,7 @@ const (
 	DefaultIgnitionTemplateSnippets string = "./template_snippets/ignition"
 	DefaultDNSMasq                  string = "/usr/sbin/dnsmasq"
 	DefaultImagesCacheDir           string = "./images"
+	DefaultFilesDir                 string = "./files"
```

Comment:
this is the new files dir, in mayu its mounted similarly like `templates` and `template_snippets`

Reply:
what about the old templates then? can you also remove `DefaultFirstStageScript` and `DefaultLastStageCloudconfig`?

Reply:
`files` dir is there for files not for templates so we need both, but yes I will be removing all the stuff like `DefaultFirstStageScript` in the clean PR which will follow after this one.